civihr_employee_portal/js/reports.js

```diff
@@ -919,6 +919,67 @@
                         }
 
                         this.$DOM.sections.dataWrapper.fadeIn();
+                        var $DOM = this.$DOM;
+
+                        $.ajax({
+                            'type': 'GET',
```

Comment:
Don't use strings for object literals key names unless it's necessary, rewrite as
```js
$.ajax({
    type: 'GET',
    url: this.getJsonUrl() + ( typeof path !== 'undefined' ? path : '' ),
    success: function () {
    }
});
```